cmd/utils/nodecmd/flags_test.go

```diff
@@ -375,39 +375,39 @@ var flagsWithValues = []struct {
 		errors:      []int{ErrorInvalidValue, NonError, ErrorInvalidValue},
 	},
 	{
-		flag:     "--rpc",
+		flag:     "--http",
 		flagType: FlagTypeBoolean,
 	},
 	{
-		flag:        "--rpcaddr",
+		flag:        "--http.addr",
```

Comment:
It is minor, but it is better to leave rpc tests and add http tests instead of substitution.